packages/mux-player/src/index.ts

```diff
@@ -379,7 +386,7 @@ class MuxPlayerElement extends VideoApiElement {
       // - native fullscreen on iPhones
       return (
         this.streamType &&
-        [StreamTypes.LIVE, StreamTypes.LL_LIVE].includes(this.streamType as any) &&
+        [StreamTypes.LIVE].includes(this.streamType as any) &&
```

Comment:
lol this and below should probably just be a strict equality check... will refactor.